.github/workflows/gitpod-check.yml

```diff
@@ -0,0 +1,15 @@
+name: Verify correct tools are installed in GitPod image
+on:
+  schedule:
+    # weekdays only
+    - cron: '0 16 * * 1-5'
+
+jobs:
+  gitpod-image-check:
+    name: Check the required tooling exists
+    uses: CorndelWithSoftwire/DevOps-Course-Workflows/.github/workflows/docker-image-tooling-check.yml@main
+    with:
+      docker-image: gitpod/workspace-dotnet:latest
+      tools-to-check: dotnet@7 node npm docker
+    secrets:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
Hmmm now that I think about it wouldn't this be better placed in the tutors repo? 
- This could potentially disrupt a learner trying to use GitHub Actions for the first time if a workflow file is already present.